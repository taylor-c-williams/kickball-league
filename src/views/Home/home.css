.kickyball {
	position: relative;
	top: 30%;
	z-index: 0;
}

.kickyball-header {
	position: relative;
	z-index: 1;
}

.kickyball-container {
	display: flex;
	justify-content: center;
	align-items: center;
}
