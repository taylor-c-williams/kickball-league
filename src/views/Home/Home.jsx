import { Link } from 'react-router-dom';
import './home.css';

export default function Home() {
	return (
		<div>
			<section className="kickyball-container">
				<header className="kickyball-header">
					<h1>Kickball League Directory! </h1>
				</header>
				<p className="kickyball">
					<img
						src="https://i5.walmartimages.com/asr/7cd32c12-dfff-4e90-983e-9dd712f11222.019e16ba07684ed951902a359a0f5504.jpeg"
						alt="kickball with adorable turtle printed on it"
					/>
					{/* another incredibly cute kickball in case I or anyone else need it:
			https://m.media-amazon.com/images/I/51vPSKT9L7L._AC_SX425_.jpg */}
				</p>
			</section>
		</div>
	);
}
