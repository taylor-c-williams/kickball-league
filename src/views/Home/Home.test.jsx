import { render } from '@testing-library/react';
import Home from './Home';

it('renders the Home view accurately', () => {
	const { container } = render(<Home />);
	expect(container).toMatchSnapshot();
});
