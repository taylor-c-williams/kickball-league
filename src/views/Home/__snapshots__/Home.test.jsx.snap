// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Home view accurately 1`] = `
<div>
  <div>
    <section
      class="kickyball-container"
    >
      <header
        class="kickyball-header"
      >
        <h1>
          Kickball League Directory! 
        </h1>
      </header>
      <p
        class="kickyball"
      >
        <a
          class="ball"
          href="/teams"
        >
          <img
            alt="kickball with adorable turtle printed on it"
            src="https://i5.walmartimages.com/asr/7cd32c12-dfff-4e90-983e-9dd712f11222.019e16ba07684ed951902a359a0f5504.jpeg"
          />
        </a>
      </p>
    </section>
  </div>
</div>
`;
